### DISCUSSION TITLE
Running Unoplatform application on autostart CM4

### DISCUSSION THREAD

**Discussion by wafamab** [unoplatform/uno#16716] [General]
Hello, I'm trying to run a Uno Platform application on autostart with FRAMEBUFFER=/dev/fb1, but I'm encountering the following issue: 'Unhandled exception. System.InvalidOperationException: Cannot read keys when either the application does not have a console or when console input has been redirected. Try Console.Read.' Has anyone encountered this issue before?

**Comment by jeromelaban** [comment]
Thanks for the report, let's track in https://github.com/unoplatform/uno/issues/16719.
